src/harness/util.py

```diff
@@ -32,14 +32,12 @@
 from reference_models.propagation import wf_hybrid
 from reference_models.antenna import antenna
 from reference_models.geo import nlcd
-from reference_models.ppa.utils import ComputePPAProtectionPoints
-
+from reference_models.geo.utils import GridPolygon
```

Comment:
Better to use : 
```
from reference_models.geo import utils
```
And reference the function with the utils namespace:
```
utils.GridPolygon()
```
Doing so keep the code more readable as we can see where the routines come from.

Reply:
GridPolygon itself is self descriptive. utils.GridPolygon isn't much different  from GridPolygon especially in a util.py function. geo.utils.GridPolygon yes

Reply:
Up to you, it is best python practice to always use external function with a module name in front:
 - it avoids namespace clash (especially when names are simple)
 - you immediately see what is defined in the current file and what is on another file
 - you immediately see where to look at to see the details of the called function
 - you can do some advance testing/patching, for example by replacing "util.GridPolygon" with a Fake implementation that do something different for testing. By redefining util.GridPolygon to some other function, all the code will use that function. With the import of the name to your file, you cannot.
Imagine for example that you want top add logging of number of time calls and number of points of the GridPolygon to get a detailed view of how much use you make of this function in the test harness (to decide if it should be optimized further). With the single location, it is easy to replace utils.GridPolygon() with a wrapper function logging stuff. 

So because the advantages are multiple, and no real disadvantage, we have policy in Google to never import a function name directly but only a module. 

This is not mandated by PEP-8, so do as you wish: there are other places in test harness where people import names directly. I have avoided that in the reference_models/ code though.